Issue #247 Added closed source software definition to repository (#112)
ubuntu/ubuntu-project-docs: docs/how-ubuntu-is-made/concepts/glossary.md

    ```

Closed Source Software

>     Software that does not allow public access to its source code.

Reviewer: I think this is a good start, it's nice and concise. In the context of developing Ubuntu though, we might also want to add some additional details, such as that packages in the `restricted` and `multiverse` components are for closed source software, usually proprietary drivers and so on.

We might also want to point to the glossary definition for Open Source Software as well -- wdyt?

Author: had a go at that :)

    ```

Closed Source Software

>     Software that does not allow public access to its source code. In Ubuntu, closed source software can typically be found in the {term}`multiverse` and {term}`restricted` components which include third party applications and proprietary drivers.

Reviewer: ```suggestion
    Software that does not allow public access to its source code. In Ubuntu, closed source software is found in the {term}`multiverse` and {term}`restricted` components which include third party applications and proprietary drivers.
```

Reviewer: closed source software is *only* found in `restricted` and `multiverse` -- by contrast, the other two components, `main` and `universe` contain all the open source software packages.

>     See also: {term} `Open Source Software`

Reviewer: ```suggestion
    See also: {term}`Open Source Software`
```
